rmgpy/molecule/graph.pyx

```diff
@@ -427,7 +444,8 @@ cdef class Graph:
         # If we need to sort then let's also update the connecitivities so
         # we're sure they are right, since the sorting labels depend on them
         self.updateConnectivityValues()
-        self.vertices.sort(key=getVertexConnectivityValue)
+
+        self.vertices = sorted(self.vertices, key=getVertexConnectivityValue)
```

Comment:
Is making a sorted copy of the list then storing it and discarding the old one better than sorting the original list in place?